firmware/mw_dual_motor_torque_ctrl/src/dual_motor_torque_ctrl.c

```diff
@@ -568,6 +568,29 @@ void main(void)
 	// At the beginning, there are no errors
 	gErrors.all = 0;
 
+	// Enable the motors programmatically.
+	gMotorVars[HAL_MTR1].Flag_enableSys = true;  // enable the system in general
+	gMotorVars[HAL_MTR1].Flag_Run_Identify = true;  // enable the motor
+	gMotorVars[HAL_MTR2].Flag_enableSys = true;  // enable the system in general
+	gMotorVars[HAL_MTR2].Flag_Run_Identify = true;  // enable the motor
```

Comment:
For safety reasons, I would prefer to keep the motors disabled on startup as default.  You could maybe add an `#ifdef` around this so it could easily be enabled when flashing boards for testing, where the "auto start" is desired.